utils/parametric/_library_client.py

```diff
@@ -624,6 +644,23 @@ def otel_get_span_context(self, span_id: int) -> OtelSpanContext:
     def otel_flush(self, timeout: int) -> bool:
         return self._client.OtelFlushSpans(pb.OtelFlushSpansArgs(seconds=timeout)).success
 
+    def get_tracer_config(self) -> dict:
```

Comment:
Same here. We need better typing. Maintain dictionaries with undefined values can be painful